.github/mergify.yml

```diff
@@ -0,0 +1,43 @@
+pull_request_rules:
+- actions:
+    merge:
+      strict: smart
+      method: merge
+  name: Automatically merge pull requests
+  conditions:
+  - status-success=bench (8.10.2, ubuntu-latest)
+  - status-success=bench (8.6.5, ubuntu-latest)
+  - status-success=bench (8.8.4, ubuntu-latest)
+  - status-success=nix (default, macOS-latest)
+  - status-success=nix (default, ubuntu-latest)
+  - status-success=test (8.10.3, ubuntu-latest)
+  - status-success=test (8.10.3, windows-latest)
+  - status-success=test (8.10.2, ubuntu-latest)
+  - status-success=test (8.10.1, ubuntu-latest)
+  - status-success=test (8.10.1, windows-latest)
+  - status-success=test (8.8.4, ubuntu-latest)
+  - status-success=test (8.8.3, ubuntu-latest)
+  - status-success=test (8.8.2, ubuntu-latest)
+  - status-success=test (8.6.5, ubuntu-latest)
+  - status-success=test (8.6.5, windows-latest)
+  - status-success=test (8.6.4, ubuntu-latest)
+  - status-success=test (windows-latest, 8.10.2.2)
+
+  - 'status-success=ci/circleci: cabal'
```

Comment:
can we drop this one since the cabal build is already covered by the GitHub Actions?

Reply:
yeah, maybe we should remove it entirely, but we can start not adding here

Reply:
I have removed it in #1152